.gitignore

```diff
@@ -28,3 +28,6 @@ src/shared/telemetry/clienttelemetry.d.ts
 # Generated by tests
 src/testFixtures/**/bin
 src/testFixtures/**/obj
+
+# raw js webviews
+compiledWebviews
```

Comment:
would this not make sense in `dist/` somewhere?

Reply:
It was separate previously because it was a part of a different webpacking job. Can probably be merged now.